test/spec/unit/pbjs_api_spec.js

```diff
@@ -1306,6 +1319,22 @@ describe('Unit: Prebid Module', function () {
       assert.ok(logErrorSpy.calledWith(error), 'expected error was logged');
       utils.logError.restore();
     });
+
+    it('should add alias to registry', () => {
+      const bidderCode = 'appnexusAst';
+      const alias = 'testalias';
+      $$PREBID_GLOBAL$$.aliasBidder(bidderCode, alias);
+      expect(adaptermanager.bidderRegistry).to.have.property(alias);
+    });
+
+    it('should add alias to registry when original adapter is using bidderFactory', function() {
+      const thisSpec = Object.assign(spec, { supportedMediaTypes: ['video'] });
+      registerBidder(thisSpec);
+      const alias = 'aliasBidder';
+      $$PREBID_GLOBAL$$.aliasBidder(CODE, alias);
```

Comment:
Try to avoid `$$PREBID_GLOBAL$$` in tests (see #1508).

Prebid-specifics aside... strong unit tests cover the smallest amount of code possible. All your changes are in `adapterManager`... so it's much better if your tests import the `adapterManager` and call the functions directly. The `$$PREBID_GLOBAL$$.aliasBidder` function is at a higher level, and would be better tested by stubbing the `adapterManager` with `sinon` and making sure the methods get called with the right args. 

On a less important note... you could probably write these in `test/spec/unit/adapterManager_spec.js`. This file is much larger than it should be already.